warehouse/query-core/src/main/java/datawave/query/jexl/functions/EvaluationPhaseFilterFunctions.java

```diff
@@ -1643,7 +1643,9 @@ public static boolean compare(Object field, String operator, String compareMode,
             }
         }
         
-        return compareFields(fields, operator, CompareFunctionValidator.Mode.valueOf(compareMode) == CompareFunctionValidator.Mode.ANY, fields2);
+        boolean matchesAny = compareMode.equalsIgnoreCase("any");
```

Comment:
why can't we use the enum here for comparison?

Reply:
fixed in next commit